mozilla_schema_generator/configs/glean.yaml

```diff
@@ -95,24 +95,53 @@ metrics:
       send_in_pings:
         not:
           - glean_ping_info
-      type: labeled_rate
+      type: not_a_real_type
+
+  # Metric types affected by bug 1737656;
+  # The jwe, labeled_rate, url, and text types were deployed to BigQuery tables
+  # with incorrect structure, so we've had to rename these types in BQ tables;
+  # we use a "2" suffix for the fields in BQ.
   jwe:
     match:
       bug_1737656_affected: true
       send_in_pings:
         not:
           - glean_ping_info
-      type: jwe
+      type: not_a_real_type
+  labeled_rate2:
```

Comment:
I've done enough testing to verify this is not sufficient to allow the rename. The code still expects the generic glean schema to contain these fields.

Reply:
One thing I wonder: Why was `labeled_rate` affected here? That's not a thing we support anyway (afaik).

Reply:
Both `labeled_rate` and `jwe` were deployed to schemas but never supported in the SDK. I figure we might as well make the changes for all 4 affected types, even though we may never have a `labeled_rate` or `jwe` metric defined. As long as no apps define these metrics, `jwe2` and `labeled_rate2` will never actually appear in schemas.

Reply:
Thanks.
And I take it that `rate` was unaffected because it was part of the initial list of metrics, without ever having been implemented (now I wonder if rates do properly work 🤔)

Reply:
Yes, the failure mode historically has been if a new metric is added to the glean JSON schema without being added to the `configs/glean.yaml` in this repository. AFAIK, `rate` was correctly in place here and so was never affected by the incorrect schema deploy issue.